### DISCUSSION TITLE
Error 400

### DISCUSSION THREAD

**Discussion by kaiquemancoso** [cjbarrie/academictwitteR#199] [Q&A] [answered]
Could anyone help me? I'm having this error:

Error in make_query(url = endpoint_url, params = params, bearer_token = bearer_token,  : 
  something went wrong. Status code: 400

```
tweetsvoto <- get_all_tweets(
    query = "voto impresso",
    start_tweets = "2021-06-27T00:00:00Z",
    end_tweets = "2021-07-17T00:00:00Z",
    n = 500000,
    data_path = "data/",
    bind_tweets = FALSE,
    country = "Brazil",
    )
```

**Comment by shmuhammadd** [comment, marked as answer]
@kaiquemancoso   

You need to correct two things.   

1. Look at the country code [here](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) , for Brazil, it is two digit `br`.

2. You need to remove the last comma after last instructions. E.g you wrote  ` country = "Brazil",`  

 The instruction below works.  


```
tweetsvoto <- get_all_tweets(
     query = "voto impresso",
     start_tweets = "2021-06-27T00:00:00Z",
     end_tweets = "2021-07-17T00:00:00Z",
     n = 5,
     data_path = "helpp",
     bind_tweets = FALSE,
     country = "br" 
 )
```

**Reply by kaiquemancoso** [reply]
Awesome! Thank you so much. It worked!
